src/styles/blocks/phone.scss

```diff
@@ -0,0 +1,21 @@
+.phone {
+  display: flex;
+  gap: 20px;
+  align-items: center;
+
```

Comment:
to remove underline you need to add text-decoration: none to phone class because it comes from tag a